app/javascript/responseTemplates/responseTemplates.js

```diff
@@ -0,0 +1,249 @@
+/* eslint-disable no-alert */
+/* eslint-disable no-restricted-globals */
+
+export function toggleTemplateTypeButton(form, e) {
+  const { targetType } = e.target.dataset
+  const activeType = targetType === 'personal' ? 'moderator' : 'personal';
+  e.target.classList.toggle('active');
+  form.querySelector(`.${activeType}-template-button`).classList.toggle('active');
+
+  form.querySelector(`.${targetType}-responses-container`).classList.toggle('hidden');
+  form.querySelector(`.${activeType}-responses-container`).classList.toggle('hidden');
+}
+
+function buildHTML(response, typeOf) {
+  if (response.length === 0 && typeOf === 'personal_comment') {
+    return `
+<div class="mod-response-wrapper mod-response-wrapper-empty">
+  <p>🤔... It looks like you don't have any templates yet.</p>
+  <p>Create templates to quickly answer FAQs or store snippets for re-use.</p>
+</div>
+`;
```

Comment:
Technically breaking a CodeClimate rule here (35 of 25 line limit). Wondering if there's a way to extract this out and move it outside of the function?